spec/numberformat.html

```diff
@@ -1594,11 +1696,17 @@ <h1>ToIntlMathematicalValue ( _value_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-getunsignedroundingmode" aoid="GetUnsignedRoundingMode">
-      <h1>GetUnsignedRoundingMode ( _roundingMode_, _isNegative_ )</h1>
-      <p>
-        The abstract operation GetUnsignedRoundingMode returns the rounding mode that should be applied to the absolute value of a number to produce the same result as if _roundingMode_, one of the String values in the *Identifier* column of <emu-xref href="#table-intl-rounding-modes"></emu-xref>, were applied to the signed value of the number (negative if the boolean _isNegative_ is *true*, or positive otherwise). The following steps are taken:
-      </p>
+    <emu-clause id="sec-getunsignedroundingmode" type="abstract operation">
+      <h1>
+        GetUnsignedRoundingMode (
+          _roundingMode_: a String,
+          _isNegative_: a Boolean,
+        ): a specification type from the *Unsigned Rounding Mode* column of <emu-xref href="#table-intl-unsigned-rounding-modes"></emu-xref>
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It returns the rounding mode that should be applied to the absolute value of a number to produce the same result as if _roundingMode_, one of the String values in the *Identifier* column of <emu-xref href="#table-intl-rounding-modes"></emu-xref>, were applied to the signed value of the number (negative if the boolean _isNegative_ is *true*, or positive otherwise).</dd>
+      </dl>
       <emu-alg>
         1. If _isNegative_ is *true*, return the specification type in the third column of <emu-xref href="#table-intl-unsigned-rounding-modes"></emu-xref> where the first column is _roundingMode_ and the second column is "negative".
         1. Else, return the specification type in the third column of <emu-xref href="#table-intl-unsigned-rounding-modes"></emu-xref> where the first column is _roundingMode_ and the second column is "positive".
```

Comment:
```suggestion
        1. If _isNegative_ is *false*, let _sign_ be “positive”; else let _sign_ be “negative”.
        1. Return the specification type in the Unsigned Rounding Mode column of <emu-xref href="#table-intl-unsigned-rounding-modes"></emu-xref> for the row where the value in the Identifier column is _roundingMode_ and the value in the Sign column is _sign_.
```

Reply:
Fixed. (Values in the "Sign" column were changed to spec enums.)